man/systemd.special.xml

```diff
@@ -328,10 +328,11 @@
       <varlistentry>
         <term><filename>kbrequest.target</filename></term>
         <listitem>
-          <para>systemd starts this target whenever Alt+ArrowUp is
-          pressed on the console. This is a good candidate to be
-          aliased (symlinked) to
-          <filename>rescue.target</filename>.</para>
+          <para>systemd starts this target whenever Alt+ArrowUp is pressed on the
+          console. It may be useful to alias this unit to
+          <filename>debug-shell.service</filename> as a last ditch debugging
+          technique. (Note that this is for <emphasis>debugging</emphasis>, and
+          leaves the machine vulnerable to local attackers).</para>
```

Comment:
I think the language here should be stronger, or it shouldn't be documented this way at all. For debugging a reproducible hang one can just boot with `systemd.debug-shell`, and debugging a non-reproducible hang would just mean to leave this on permanently - this is quite a gaping local root hole.